src/redux/eventsReducer.js

```diff
@@ -0,0 +1,39 @@
+import axios from 'axios';
+
+// globals
+const dict = {};
+const initialState = {};
+const URL = 'https://kontests.net/api/v1/all';
+
+// Actions
+const FETCH_EVENTS = 'FETCH_EVENTS';
```

Comment:
- [Optional] As a good practice, always try to scope the action names based on the feature they belong to. This helps when debugging more complex applications.